tests/utils/test_utils.py

```diff
@@ -222,90 +219,110 @@ def mock_visualize_mph(mph_df, out_dir, return_plot=True, regression: bool = Fal
         fig = plt.figure()
         ax1 = fig.add_subplot(111)
         ax1.scatter(mph_df["cum_total_count"], mph_df["MPH"])
-        ax2 = ax1.twiny()
```

Comment:
We should leave this in, if I remember correctly the QC visualization needs this.

Reply:
It's for mph and this is just a mock plotting for watcher testing. None of the plot ticks are correctly set anyways.